src/FastAlmostBandedMatrices.jl

```diff
@@ -3,13 +3,13 @@ module FastAlmostBandedMatrices
 import PrecompileTools: @recompile_invalidations, @setup_workload, @compile_workload
 
 @recompile_invalidations begin
-    using ArrayInterface, ArrayLayouts, BandedMatrices, ConcreteStructs, LazyArrays,
-        LinearAlgebra, MatrixFactorizations, Reexport
+	using ArrayInterface, ArrayLayouts, BandedMatrices, ConcreteStructs, LazyArrays,
+		LinearAlgebra, MatrixFactorizations, Reexport
```

Comment:
You editor changed all the spaces into tabs

Reply:
:((( Maybe my formatter is badly setup...